### PR TITLE
Add shortcut to toggle overlay

### LINKED ISSUE #144: Toggle or Relocate Play Bar
The play bar at the bottom of the recorded screen obstructs an element I need to access.  Is it possible to relocate or toggle on/off the play bar?

### PR DESCRIPTION
**Pull request by ianaya89**
# Add shortcut to toggle overlay

## Description
Add shortcut to toggle overlay and re-add screenshot shortcuts:
- `ctrl + k`: Toggle overlay
- `ctrl + shitf + F`: Take full page screenshot
- `ctrl + shitf + E`: Take element screenshot

hr-demo

> Resolves #144 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project. `npm run lint` passes with no errors.
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes. `npm run test` passes with no errors.

### CODE REVIEW

**Comment by tnolet** [comment]
@ianaya89 can you add a screenshot?

**Comment by ianaya89** [comment]
@tnolet done!

File: README.md [hunk 1]
```diff
@@ -78,6 +78,12 @@ This project builds on existing open source projects (see [Credits](#-credits))
 
 5. Click Pause when you want to navigate without recording anything. Hit Resume to continue recording.
 
+### ⌨️ Shortcuts
+
+- `ctrl + k`: Toggle overlay
+- `ctrl + shitf + F`: Take full page screenshot
```
Comment (tnolet) [inline README.md:84]: nice typo...`shit`

**Review by tnolet — CHANGES_REQUESTED** [review]
😅 almost!

**Review by maxigimenez — APPROVED** [review]
LGTM 👍 !

**Pull request merged**